crates/fuel-core/src/query/message.rs

```diff
@@ -278,7 +278,11 @@ fn message_receipts_proof<T: MessageProofData + ?Sized>(
     // Get the message receipts from the block.
     let leaves: Vec<Vec<Receipt>> = message_block_txs
         .iter()
-        .map(|id| database.receipts(id))
```

Comment:
Maybe we also need to update another place with `database.receipts` function call. And remove `receipts` function at all=)

<img width="1070" alt="image" src="https://github.com/user-attachments/assets/62deaa3d-72c6-473a-9d01-a6eea29a9692">